TaskfMRIAnalysis/scripts/makeSubjectTaskSummary.sh

```diff
@@ -0,0 +1,363 @@
+opts_AddOptional '--lowpassfilter' 'TemporalSmoothing' 'integer' "Apply *additional* lowpass filter (in seconds) to time series and task design. This is above and beyond temporal filter applied during preprocessing. Low pass filter is generally not advised for Task fMRI analyses. Default=NONE" 'NONE'
+opts_AddOptional '--procstring' 'ProcSTRING' 'string' "String value in filename of time series image, specifying the additional processing that was previously applied (e.g., FIX-cleaned data with 'hp2000_clean' in filename). Default=NONE" 'NONE'
+opts_AddOptional '--lowresmesh' 'LowResMesh' 'integer' "Value (in mm) that matches surface resolution for fMRI data. Default=32, which is appropriate for HCP minimal preprocessing pipeline outputs" '32'
+opts_AddOptional '--grayordinatesres' 'GrayordinatesResolution' 'number' "Value (in mm) that matches value in 'Atlas_ROIs' filename; Default='2', which is appropriate for HCP minimal preprocessing pipeline outputs" '2'
+opts_AddOptional '--regname' 'RegName' 'RegName' "Name of surface registration technique. Default=NONE, which will use the default (MSMSulc) surface registration." 'NONE'
+opts_AddOptional '--vba' 'VolumeBasedProcessing' 'YES/NO' "Default=NO. CAUTION: Only use YES if you want unconstrained volumetric blurring of your data, otherwise set to NO for faster, less biased, and more senstive processing (grayordinates results do not use unconstrained volumetric blurring and are always produced)" 'NO'
+opts_AddOptional '--parcellation' 'Parcellation' 'ParcellationName' "Name of parcellation scheme to conduct parcellated analysis. Default=NONE, which will perform dense analysis instead. Non-greyordinates parcellations are not supported because they are not valid for cerebral cortex.  Parcellation supersedes smoothing (i.e. no smoothing is done)" 'NONE'
+opts_AddOptional '--parcellationfile' 'ParcellationFile' '/path/to/dlabel' "Absolute path to the parcellation dlabel file. Default=NONE" 'NONE'
+
+opts_ParseArguments "$@"
+
+# if LevelOnefsfNames is blank, set equal to LevelOnefMRINames
+[ -z "$LevelOnefsfNames" ] && LevelOnefsfNames=${LevelOnefMRINames}
+# if LevelTwofsfName is blank, set equal to LevelTwofMRIName
+[ -z "$LevelTwofsfName" ] && LevelTwofsfName=${LevelTwofMRIName}
+
+#display the parsed/default values
+opts_ShowValues
+
+
+# ------------------------------------------------------------------------------
+#  Verify required environment variables are set and log value
+# ------------------------------------------------------------------------------
+
+if ((pipedirguessed))
+then
+    log_Err_Abort "HCPPIPEDIR is not set, you must first source your edited copy of Examples/Scripts/SetUpHCPPipeline.sh"
+fi
+
+log_Msg "Platform Information Follows: "
+uname -a
+
+opts_ShowVersionIfRequested $@
+${HCPPIPEDIR}/show_version
```

Comment:
if you run show_version unconditionally, it doesn't make sense to optionally show version another way, and `opts_ShowVersionIfRequested` is a legacy function, please don't use it.  If we really want all pipeline scripts to have a hardcoded `--version` option that shows the overall version of the entire pipelines repo (seems a bit silly to me, which is why I didn't do it yet), that can be added to `newopts.shlib` without needing any code in each script to activate it.

Reply:
This is addressed in next commit.